Methods.java

```diff
@@ -0,0 +1,39 @@
+package Assignment_3;
+
+public class Methods {
+    public static int sumOfDigits(int number) {
+        if(number >= 0 && number <10){
+            return number;
+        }
+
+        while (number >= 10) {
+            int sum = 0;
+            while (number > 0) {
+                sum += number % 10;
+                number /= 10;
+            }
+            number = sum;
+        }
+        return number;
+    }
+
+    public static String findConsecutiveSums(int number) {

```

Comment:
describe each method in brief through comments including short definition , input parameters and return values .